docs/OV_Runtime_UG/multi_device.md

```diff
@@ -137,186 +115,46 @@ You can use the [configure devices](supported_plugins/config_properties.md) to q
 
-Following the OpenVINO™ convention of labeling devices, the Multi-Device plugin uses the name "MULTI". The only configuration option for the Multi-Device plugin is a prioritized list of devices to use:
+## Using the Multi-Device with OpenVINO Samples and Benchmarking Performance
 
-| Parameter name | Parameter values | Default | Description |
-| -------------- | ---------------- | --- | --- |
-| "MULTI_DEVICE_PRIORITIES" | comma-separated device names with no spaces | N/A | Prioritized list of devices |
-
-You can set the configuration directly as a string, or use the metric key `MULTI_DEVICE_PRIORITIES` from the `multi/multi_device_config.hpp` file, which defines the same string.
-
-#### The Three Ways to Specify Devices Targets for the MULTI plugin
-
-* Option 1 - Pass a Prioritized List as a Parameter in ie.load_network()
-
-@sphinxdirective
-
-.. tab:: Python
-
-    .. doxygensnippet:: docs/snippets/ov_multi.py
-       :language: python
-       :fragment: [Option_1]
-
-@endsphinxdirective
-
-* Option 2 - Pass a List as a Parameter, and Dynamically Change Priorities during Execution
-   Notice that the priorities of the devices can be changed in real time for the compiled model:
-
-@sphinxdirective
-
-.. tab:: Python
-
-    .. doxygensnippet:: docs/snippets/ov_multi.py
-       :language: python
-       :fragment: [Option_2]
-
-@endsphinxdirective
-
-* Option 3 - Use Explicit Hints for Controlling Request Numbers Executed by Devices
-   There is a way to specify the number of requests that Multi-Device will internally keep for each device. If the original app was running 4 cameras with 4 inference requests, it might be best to share these 4 requests between 2 devices used in the MULTI. The easiest way is to specify a number of requests for each device using parentheses: “MULTI:CPU(2),GPU(2)” and use the same 4 requests in the app. However, such an explicit configuration is not performance-portable and not recommended. The better way is to configure the individual devices and query the resulting number of requests to be used at the application level. See [Configuring the Individual Devices and Creating the Multi-Device On Top](#configuring-the-individual-devices-and-creating-the-multi-device-on-top).
-
-
-### Enumerating Available Devices
-The OpenVINO Runtime API features a dedicated methods to enumerate devices and their capabilities. See the [Hello Query Device Python Sample](../../samples/python/hello_query_device/README.md). This is example output from the sample (truncated to device names only):
+To see how the Multi-Device execution is used in practice and test its performance, take a look at OpenVINO™ samples. All samples supporting the `-d` command-line option (which stands for “device”) will accept the plugin out-of-the-box. The Benchmark Application will be a perfect place to start – it presents the optimal performance of the plugin without the need for additional settings, like the number of requests or CPU threads. 
```

Comment:
"will accept the plugin out-of-the-box" is confusing... we are usually talking about the "devices", rather than "plugins".
the benchmark_app is today the on;ly option, as the rest of the samples are API examples, and the MULTI is about  the performance, whihc only the benchamrk_app reports.

Reply:
I guess "will accept the MULTI mode natively" is a better choice of words. Is it ok? 
But does it mean we should not mention the '-d" option? If so, we can remove the whole sentence.